.vscode/tasks.json

```
@@ -64,7 +64,7 @@
       "label": "Code Coverage",
       "detail": "Generate code coverage report for a given integration.",
       "type": "shell",
-      "command": "pytest ./tests/components/${input:integrationName}/ --cov=homeassistant.components.${input:integrationName} --cov-report term-missing",
+      "command": "pytest ./tests/components/${input:integrationName}/ --cov=homeassistant.components.${input:integrationName} --cov-report term-missing --durations-min=5 --durations=0",
```

I think that even 1 second is quite big for a unit test.

```suggestion
      "command": "pytest ./tests/components/${input:integrationName}/ --cov=homeassistant.components.${input:integrationName} --cov-report term-missing --durations-min=1 --durations=0",
```

---

I agree, especially when running a specific suite, these should be painfully be presented with a low threshold.

---

Updated to 1s